devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy

```diff
@@ -0,0 +1,162 @@
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+	definition(name: "Z-Wave Motion Light", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.light", mmnm: "SmartThings", vid: "generic-motion-light") {
+		capability "Switch"
+		capability "Motion Sensor"
+		capability "Illuminance Measurement"
+		capability "Sensor"
+		capability "Health Check"
+		capability "Configuration"
+
+		fingerprint mfr: "0060", prod: "0012", model: "0001", deviceJoinName: "Everspring Outdoor Floodlight"
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name:"switch", type: "lighting", width: 1, height: 1, canChangeIcon: true) {
+			tileAttribute("device.switch", key: "PRIMARY_CONTROL") {
+				attributeState("on", label:'${name}', action:"switch.off", icon:"st.lights.philips.hue-single", backgroundColor:"#00a0dc", nextState:"turningOff")
+				attributeState("off", label:'${name}', action:"switch.on", icon:"st.lights.philips.hue-single", backgroundColor:"#ffffff", nextState:"turningOn")
+				attributeState("turningOn", label:'${name}', action:"switch.off", icon:"st.lights.philips.hue-single", backgroundColor:"#00a0dc", nextState:"turningOff")
+				attributeState("turningOff", label:'${name}', action:"switch.on", icon:"st.lights.philips.hue-single", backgroundColor:"#ffffff", nextState:"turningOn")
+			}
+		}
+		valueTile("motion", "device.motion", decoration: "flat", width: 2, height: 2) {
+			state("active", label: 'motion', icon: "st.motion.motion.active", backgroundColor: "#00A0DC")
+			state("inactive", label: 'no motion', icon: "st.motion.motion.inactive", backgroundColor: "#CCCCCC")
+		}
+		valueTile("illuminance", "device.illuminance", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "illuminance", label: '${currentValue} lux', backgroundColors: [
+					[value: 40, color: "#999900"],
+					[value: 100, color: "#CCCC00"],
+					[value: 300, color: "#FFFF00"],
+					[value: 500, color: "#FFFF33"],
+					[value: 1000, color: "#FFFF66"],
+					[value: 2000, color: "#FFFF99"],
+					[value: 10000, color: "#FFFFCC"]
+			]
+		}
+
+		main "switch"
+		details(["switch", "motion", "illuminance"])
+	}
+}
+
+def initialize() {
+	sendEvent(name: "checkInterval", value: 2 * 4 * 60 * 60 + 24 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	sendEvent(name: "motion", value: "inactive", displayed: false)
+}
+
+def installed() {
+	initialize()
+}
+
+def updated() {
+	initialize()
+}
+
+def configure() {
+	[
+			secure(zwave.notificationV3.notificationGet(notificationType: 0x07)),
+			secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x03)),
+			secure(zwave.switchBinaryV1.switchBinaryGet()),
+			secure(zwave.configurationV1.configurationSet(parameterNumber: 3, size: 2, scaledConfigurationValue: 600)) //enables illuminance report every 10 minutes
```

Comment:
This is device specific... should probably have a check for that. Just thinking out loud for when we add more devices to this DTH.